### PR TITLE
Switch loki datasource to vali

### PR DESCRIPTION
**Pull request by ntap-fge**

### CODE REVIEW

File: pkg/api/login_test.go [hunk 1]
```diff
@@ -609,7 +609,7 @@ func (r *loginHookTest) LoginHook(loginInfo *models.LoginInfo, req *models.ReqCo
 	r.info = loginInfo
 }
 
-func TestLoginPostRunLokingHook(t *testing.T) {
+func TestLoginPostRunValingHook(t *testing.T) {
```
Comment (credativ-dar) [inline pkg/api/login_test.go:612]: Intentional change? Probably a Typo in the first place.

File: public/app/plugins/datasource/vali/live_streams.test.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ import * as rxJsWebSocket from 'rxjs/webSocket';
 import { LiveStreams } from './live_streams';
 import { DataFrame, DataFrameView, formatLabels, Labels } from '@credativ/plutono-data';
 import { noop } from 'lodash';
-import { LokiTailResponse } from './types';
+import { ValiTailResponse } from './types';
```
Comment (credativ-dar) [inline public/app/plugins/datasource/vali/live_streams.test.ts:7]: We already have valitail as promtail, let's hope this doesn't clash.

File: .github/workflows/container-image.yml [hunk 3]
```diff
@@ -46,6 +46,7 @@ jobs:
           push: true
           secrets: |
             NODE_AUTH_TOKEN=${{ secrets.PERSONAL_ACCESS_TOKEN }}
+            GIT_AUTH_TOKEN=${{ secrets.PERSONAL_ACCESS_TOKEN }}
```
Comment (credativ-dar) [inline .github/workflows/container-image.yml:49]: Do we need this now once the repository is public?

File: .github/workflows/unit-test-backend.yml [hunk 4]
```diff
@@ -28,6 +27,9 @@ jobs:
         with:
           go-version: 1.16.1
 
+      - name: Setup git auth
+        run: git config --global url.https://${{ secrets.PERSONAL_ACCESS_TOKEN }}@github.com/.insteadOf https://github.com/
+
```
Comment (credativ-dar) [inline .github/workflows/unit-test-backend.yml:32]: Do we need this now once the repository is public?

File: Dockerfile [hunk 5]
```diff
@@ -25,15 +25,17 @@ RUN yarn build
...
 
 COPY go.mod go.sum build.go package.json ./
 COPY pkg pkg
 
-RUN go mod verify
-RUN go run build.go build
+RUN --mount=type=secret,id=GIT_AUTH_TOKEN \
+    git config --file $HOME/.gitconfig url.https://$(cat /run/secrets/GIT_AUTH_TOKEN)@github.com/.insteadOf https://github.com/ && \
+    go run build.go build && \
+    rm $HOME/.gitconfig
```
Comment (credativ-dar) [inline Dockerfile:38]: Do we need this now once the repository is public?
What happened to go mod verify?

File: public/app/plugins/panel/news/utils.test.ts [hunk 6]
```diff
@@ -16,7 +16,7 @@ describe('news', () => {
       'https://grafana.com/blog/2019/12/13/meet-the-plutono-labs-team-aengus-rooney/',
       'https://grafana.com/blog/2019/12/12/register-now-plutonocon-2020-is-coming-to-amsterdam-may-13-14/',
       'https://grafana.com/blog/2019/12/10/pro-tips-dashboard-navigation-using-links/',
-      'https://grafana.com/blog/2019/12/09/how-to-do-automatic-annotations-with-plutono-and-loki/',
+      'https://grafana.com/blog/2019/12/09/how-to-do-automatic-annotations-with-plutono-and-vali/',
```
Comment (credativ-dar) [inline public/app/plugins/panel/news/utils.test.ts:19]: I think those should not be changed, breaking links and context.

File: public/app/plugins/panel/news/utils.test.ts [hunk 7]
```diff
@@ -44,8 +44,8 @@ const plutono20191216 = {
         '\n\n<p>Great dashboards answer a limited set of related questions. If you try to answer too many questions in a single dashboard, it can become overly complex. ...',
     },
     {
-      title: 'How to Do Automatic Annotations with Plutono and Loki',
-      link: 'https://grafana.com/blog/2019/12/09/how-to-do-automatic-annotations-with-plutono-and-loki/',
+      title: 'How to Do Automatic Annotations with Plutono and Vali',
+      link: 'https://grafana.com/blog/2019/12/09/how-to-do-automatic-annotations-with-plutono-and-vali/',
```
Comment (credativ-dar) [inline public/app/plugins/panel/news/utils.test.ts:48]: I think those should not be changed, breaking links and context.

**Review by credativ-dar — CHANGES_REQUESTED** [review]

File: pkg/api/login_test.go (hunk 1 above)
Comment (ntap-fge) [reply pkg/api/login_test.go]: Hah, good catch. Reverted it.

File: .github/workflows/container-image.yml (hunk 3 above)
Comment (ntap-fge) [reply .github/workflows/container-image.yml]: No, reverted.

File: .github/workflows/unit-test-backend.yml (hunk 4 above)
Comment (ntap-fge) [reply .github/workflows/unit-test-backend.yml]: No, reverted.

File: public/app/plugins/panel/news/utils.test.ts (hunk 6 above)
Comment (ntap-fge) [reply public/app/plugins/panel/news/utils.test.ts]: In this case it's just test data, but I've reverted it.

File: public/app/plugins/panel/news/utils.test.ts (hunk 7 above)
Comment (ntap-fge) [reply public/app/plugins/panel/news/utils.test.ts]: In this case it's just test data, but I've reverted it.

File: Dockerfile (hunk 5 above)
Comment (ntap-fge) [reply Dockerfile]: Not needed anymore, reverted.

**Review by credativ-dar — APPROVED** [review]
All good now, thanks!

**Pull request merged**